.github/workflows/build-gh.yml

```diff
@@ -42,11 +45,12 @@ jobs:
     needs:
       - set-vars
       - start-runner
-    uses: i-dot-ai/i-dot-ai-core-github-actions/.github/workflows/build-docker.yml@main
+    uses: i-dot-ai/i-dot-ai-core-github-actions/.github/workflows/build-docker.yml@docker-cache
```

Comment:
Change the branch reference back to main

Reply:
I have fixed it.